packages/gestalt-datepicker/src/DatePicker.js

```diff
@@ -110,6 +173,8 @@ const DatePickerWithForwardRef: React$AbstractComponent<Props, HTMLDivElement> =
       (rangeSelector === 'end' && !rangeStartDate)
     ) {
       if (nextRef && submitted) {
+        // $FlowFixMe[prop-missing]
```

Comment:
Hm…are you sure the new type for `nextRef` is correct? Can we avoid these suppressions?